root/static/scripts/edit/components/FormRowTextList.js

```diff
@@ -10,15 +10,18 @@
 import mutate from 'mutate-cow';
 import React from 'react';
 
-import {pushField} from '../utility/pushField.js';
+import {pushCompoundField} from '../utility/pushField.js';
 
 import AddButton from './AddButton.js';
 import FieldErrors from './FieldErrors.js';
 import FormLabel from './FormLabel.js';
 import FormRow from './FormRow.js';
 import RemoveButton from './RemoveButton.js';
 
-type StateT = RepeatableFieldT<FieldT<string>>;
+type StateT = RepeatableFieldT<CompoundFieldT<{
+  +removed: FieldT<boolean>,
+  +value: FieldT<string>,
+}>>;
```

Comment:
Super minor, but this is starting to look like the kind of type that deserves a name and to live somewhere reusable, since it will be needed I assume once we have other forms also typed in Flow like we do for `GenreFormT` rn. Either the Repeatable or at least the Compound inside with this values, especially if we're planning to have most forms use this rather than the previous style?